examples/index.html

```diff
@@ -3,21 +3,74 @@
 <head>
     <meta charset="utf-8">
     <title>HW provider example</title>
+    <style>
+        button {
+            margin-top: 5px;
+            margin-bottom: 5px;
+        }
+    </style>
 </head>
 
 <body>
     <div>
-        <button onclick="main()">Run example</button>
+        <button onclick="init()">Initialize</button>
+    </div>
+    <div>
+        <input value="0" type="number" min="0" name="addressIndexForLogin" />
+        <button onclick="login()">Login</button>
+    </div>
+    <div>
+        <input value="0" type="number" min="0" name="addressIndexForSetAddress" />
+        <button onclick="setAddress()">Set address index</button>
+    </div>
+    <div>
+        <button onclick="signTransaction()">Sign transaction</button>
     </div>
 
     <script src="../out-examples/lib.js"></script>
     <script>
+        var provider = new lib.HWProvider();
+
+        async function init() {
+            await provider.init();
+            console.log("Initialized.");
+        }
+
+        async function login() {
+            var addressIndex = parseInt(document.getElementsByName("addressIndexForLogin")[0].value);
+            console.log("Login with addressIndex", addressIndex);
+            await provider.login({ addressIndex: addressIndex });
+            await dumpAddresses();
```

Comment:
dumpAddresses sounds a bit like dropAddresses, can be confusing a bit, maybe "logAddresses"?

Reply:
Fixed (renamed).